### PR TITLE
feat: just exit tmuxinator alias

### PR DESCRIPTION
**Pull request by okjodom**
I just want to exit tmuxinator you know

### CODE REVIEW

File: justfile [hunk 1]
```diff
@@ -81,3 +81,7 @@ format:
 # start tmuxinator with a dev federation setup
 tmuxinator:
   ./scripts/tmuxinator.sh
+
+# exit tmuxinator session
+exit-tmuxinator:
+  tmux kill-session -t fedimint-dev
```
Comment (oleonardolima) [inline justfile:87]: @okjodom I wonder if also adding a new cmd to restart (using the existing script) would also be helpful, wdyt ? 😁

Comment (okjodom) [reply justfile:87]: You can just propose that alias. Would be useful!

**Comment by dpc** [comment]
LGTM, but @Maan2003 is in charge.

**Review by Maan2003 — APPROVED** [review]
LGTM

**Review by dpc — APPROVED** [review]

**Pull request merged**